### ISSUE TITLE
Training problem.

### ISSUE THREAD

**Issue by NienfengYao** [Ashing00/Tensorflow_VGG#1]
Hi Ashing,

I'm a beginner to study ML. I reference your project Tensorflow_VGG to study VGG16 with CIFAR-10 dataset  to do training, it's great. 

For more exercise, I follow your training procedure but using another VGG16 model from [VGG in TensorFlow](https://www.cs.toronto.edu/~frossard/post/vgg16/). The exercise is [vgg16f_training_cifar10.ipynb](https://github.com/NienfengYao/TensorFlow_Study/blob/master/vgg16/vgg16_final/vgg16f_training_cifar10.ipynb). But I can't get good accuracy and still try to fine the root cause. Could you give me some hints?
Thanks a lot.

**Comment by NienfengYao** [comment]
I add batch normalization in each hidden layer. Now we can get good accuracy. But in fact, I don't know why I can not get good result if no batch normalization.
